src/cephadm/cephadmlib/daemons/smb.py

```diff
@@ -237,6 +240,28 @@ def __init__(self, ctx: CephadmContext, ident: DaemonIdentity):
         self.smb_port = 445
         logger.debug('Created SMB ContainerDaemonForm instance')
 
+    @staticmethod
+    def get_version(ctx, container_id):
+        # type: (CephadmContext, str) -> Optional[str]
```

Comment:
Please do not add new comment style annotations to the code, even if that's what older classes have.

Reply:
Done.